src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -227,6 +238,9 @@ def load_arguments(self, _):
         c.argument('storage_account', help='Name or ID of the staging storage account. The VM configuration will be restored to this storage account. See the help for --restore-to-staging-storage-account parameter for more info.')
         c.argument('restore_to_staging_storage_account', arg_type=get_three_state_flag(), help='Use this flag when you want disks to be restored to the staging storage account using the --storage-account parameter. When not specified, disks will be restored to their original storage accounts. Default: false.')
         c.argument('target_resource_group', options_list=['--target-resource-group', '-t'], help='Use this to specify the target resource group in which the restored disks will be saved')
+        c.argument('diskslist', diskslist_type)
+        c.argument('restore_only_osdisk', arg_type=get_three_state_flag(), help='Use this flag to restore only OS disks of a backed up VM.')
+        c.argument('restoredisks', arg_type=get_three_state_flag(), help='Use this flag to restore specified disks of a backed up VM.')
```

Comment:
Maybe we can add a "_" between "restore" and "disks"

Reply:
On second thoughts, this may not be required at all. The command name is already 'az backup restore restore-disks'. Here the expected arguments are either 'restore_only_osdisk' or "diskslist". Both are optional and only one of them is expected. If both of them are not given, then the service will go with default behavior. @sambitratha : Please remove the restoredisks parameter.

Reply:
Done